app/models/campaign.rb

```diff
@@ -104,6 +118,19 @@ def seller_distributor_pairs
 
   private
 
+  def mega_gam?
```

Comment:
No need to do this now, but this attribute probably will be public in the future

Reply:
yeah i'm going to address this in https://github.com/sendchinatownlove/ruby/pull/290/files